docs/quick_tutorial/hello_world.rst

```diff
@@ -56,6 +56,17 @@ Steps
    .. literalinclude:: hello_world/app.py
     :linenos:
 
+The following tree diagram shows the current structure.
+
+   .. code-block:: text
+
+    `──/path/to/project/
+        `── projects
+            `── quick_tutorial
+                │── env
+                `── hello_world
+                       `── app.py
+
```

Comment:
Move the entire section and its new heading after **Steps** and just before **Analysis**.

Also `.. code-block:: text` should be flush left with no indentation, and the code sample should be indented with 4 spaces.